### PR TITLE
Updated tags for Plug & fetch feeds in XSOAR

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5113

## Description
Were updating the tags for feeds for better searches and visibility within XSOAR.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/TwitterIOCHunter-FullDailyFeed/pack_metadata.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "categories": ["Data Enrichment & Threat Intelligence"],
     "tags": [
         "Threat Intelligence Management",
-        "Getting Started"
+        "Getting Started",
```
Comment (altmannyarden) [inline Packs/TwitterIOCHunter-FullDailyFeed/pack_metadata.json:13]: As its community supported, lets not change this pack metadata.
I also know that Twitter API changed recently (with paid version).

**Review by bakatzir — COMMENTED** [review]
Le'ts have Alon Y (and a tech writer) in the discussion as well(not the PR) on the best way to represent non-paid feeds.
Generic Feed/Free Feed/ Plug & Fetch/Getting Started/etc.

File: Packs/TwitterIOCHunter-FullDailyFeed/pack_metadata.json (hunk 1 above)
Comment (Ni-Knight) [reply Packs/TwitterIOCHunter-FullDailyFeed/pack_metadata.json:13]: Ok. but it has "Getting Started"

**Comment by Ni-Knight** [comment]
@bakatzir Non paid feeds != Plug & Fetch

Plug & Fetch == Free feed and no creds needed, i.e. you plug it and fetch IOCs (like plug & play).
Free Feed == Feed that is free (it might require creds and signup).
Getting Started == Not sure, never saw it in the documentation.

**Review by bakatzir — APPROVED** [review]
I'm approving the additions to Tests/Marketplace/approved_tags.json.

Yarden/R&D to review the other files.

File: Packs/FreeFeeds/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
+{
+    "name": "Free Feeds",
+    "description": "This content Pack helps set up free feeds (Plug & Fetch, Free with signup and Generic) available in TIM",
```
Comment (altmannyarden) [inline Packs/FreeFeeds/pack_metadata.json:3]: in or for TIM?

Comment (Ni-Knight) [reply Packs/FreeFeeds/pack_metadata.json:3]: Probably "on" is actually best here.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**